pandas/core/generic.py

```diff
@@ -2790,6 +2791,11 @@ def to_latex(self, buf=None, columns=None, col_space=None, header=True,
             By default, the value will be read from the pandas config
             module. Use a longtable environment instead of tabular. Requires
             adding a \usepackage{longtable} to your LaTeX preamble.
+        caption : str, optional
+            The LaTeX caption to be placed inside \caption{} in the output.
```

Comment:
can you add versionadded to each of these

Reply:
yes will do.  thanks.